### ISSUE TITLE
refactor: Choose better fonts?

### ISSUE THREAD

**Issue by Stefan-Radu** [dothidden/writeups#43]
* I really liked the one from Google CTF but not sure if it's easy to read. We could also have wepparate font for headings and writing

**Comment by Mathis-Dory** [comment]
I claim this issue, I will try to work on it today or tomorrow to improve the general style, I also claimed #47.

**Comment by Mathis-Dory** [comment]
@Stefan-Radu  What do you think about maybe we can use many of them for different purposes and create archetype for it for example 'hacked' font in the title for PWN category, ....
<img width="1080" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/11c1eb8c-cd5b-490a-b21b-9f1e0c0e9de7">

<img width="1112" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/af010199-a11d-400a-901b-f2d68ef1a023">
<img width="1015" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/4701824c-d59d-49e0-9754-e43042a21de7">
<img width="1061" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/ae1d97e2-269f-4496-ad00-b778903542d6">
<img width="1040" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/a21c3079-0124-476c-a881-2b5a69f1f539">
<img width="1008" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/b492c46d-c90d-4fd4-aa27-8bc2b00f8286">

**Comment by Stefan-Radu** [comment]
@Mathis-Dory I would honestly keep it simple. I think it's also a good design rule to not clutter up the site with too many fonts. Two, max 3 is usually recommended afaik, and I would suggest we stick to that recommendation.

HOWEVER, I really like what you're suggesting with the pixelated fonts. This is exactly what I was thinking as well, and I kept the idea in mind since GoogleCTF because I really liked their aesthetic. This type of font would be too much for text, so I was thinking of choosing a pixelated font for headings (h1, h2, maybe h3?) and a sans / serif font for the rest of the content.

If I manage I'll suggest some pairings as well. I'll react to the ones I like from the ones you suggested

**Comment by Stefan-Radu** [comment]
Ok nvm, it's just one image. I like the `Atari` and `Hackbot`. Did you make sure they're open fonts? I see something regarding a free trial for `Uncracked`.

**Comment by Mathis-Dory** [comment]
@Stefan-Radu They are for non commercial use so yes we can use it, I also like `Uncracked` and `Hacked`

**Comment by Stefan-Radu** [comment]
I would suggest we start proposing some pairings and get the whole team to vote. I'll give some suggestions today

**Comment by Mathis-Dory** [comment]
@Costinteo @SecioreanuStefanita  vote for your fav

**Comment by Costinteo** [comment]
I think the glitched fonts might look weird when used as regular fonts instead of in some poster or something. I have wildly different preferences for fonts, I'd rather have something very easy to read, pixel/retro fonts and glitched fonts are not that easy to read. But I really don't mind that much and you can use any of them on the site.

**Comment by Stefan-Radu** [comment]
> I think the glitched fonts might look weird when used as regular fonts instead of in some poster or something. I have wildly different preferences for fonts, I'd rather have something very easy to read, pixel/retro fonts and glitched fonts are not that easy to read. But I really don't mind that much and you can use any of them on the site.

I think we should all agree if we decide on changing the fonts. I was thinking of using a cool font for the headings (or at least the title) and an easily readable font for the text body.

**Comment by Stefan-Radu** [comment]
yeah nvm. I give up. I tried for a few hours to find something that fits, but it's taking so much time and I feel I'm getting nowhere. I find it too difficult. If someone else is interested in working on this, by all means do it!

**Comment by Mathis-Dory** [comment]
I agree a special font can be very cool for headers. I will try to find it now

**Comment by Mathis-Dory** [comment]
Here you have some preview I like

**### Modeseven (h1,h2,h3):**
<img width="939" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/d19d7d71-745b-4428-8053-fd6893769483">
<img width="1075" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/e80b9f76-fb64-4d47-9f3d-3163619dfdf0">


**### Atari (h1,h2)**

<img width="867" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/49281e0c-e42e-4399-b043-66ae00fc5a37">
<img width="908" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/2df9c3a5-0f70-4f6b-aae3-2fbebc0f226f">

**### Connection(h1,h2)**

<img width="903" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/0c993210-bd00-4c9d-bdd8-9e4e4444d295">
<img width="915" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/61f1c621-202a-48cf-ac86-ddbb446ed674">

**### Hacked(h1,h2)**

<img width="857" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/3fdd43dc-e625-4ae9-99f2-5435378de5bf">
<img width="946" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/9c6d63a6-8803-4879-a6fd-27a77c745957">

### **Connection(h1,h2) + Hacked(h3,h4,h5,h6)**

<img width="818" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/e300541f-bce4-4998-8562-3b2aef53579a">
<img width="529" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/ef24b527-d78f-4364-b33e-c30a5201e870">
<img width="857" alt="image" src="https://github.com/dothidden/writeups/assets/64088463/feb3ebd1-dab7-4108-8731-b5420c661f9c">


@Stefan-Radu @Costinteo @SecioreanuStefanita  vote if you find something cool in the above preview, maybe we can also apply special typo to link, navbar, underline text, ...

**Comment by Stefan-Radu** [comment]
Unfortunately, I don't really like any of them. This is why I also got stuck while trying fonts myself. They create inconsistency and also make the text harder to read, which is bad. While they look kinda cool, I feel like they also don't fit. Not sure what to say, or what to suggest. My gut tells me we should let it as it is right now =)))

**Comment by Mathis-Dory** [comment]
> Unfortunately, I don't really like any of them. This is why I also got stuck while trying fonts myself. They create inconsistency and also make the text harder to read, which is bad. While they look kinda cool, I feel like they also don't fit. Not sure what to say, or what to suggest. My gut tells me we should let it as it is right now =)))

So issue closed ?

**Comment by Stefan-Radu** [comment]
I you're alright with this yes. I kinda sucks that you wasted effort on this.
